pygfunction/gfunction.py

```diff
@@ -1296,7 +1306,7 @@ def solve(self, time, alpha):
                     # Energy conservation: sum([Q_b*H_b]) = sum([H_b])
                     # ---------------------------------------------------------
                     a_in, a_b = self.network.coefficients_borehole_heat_extraction_rate(
-                            self.network.m_flow_network, self.network.cp_f, self.nSegments)
+                            self.network.m_flow_network, self.network.cp_f, self.nBoreSegments)
```

Comment:
> I will work on the review soon after validating there is no conflict with the `'MIFT` boundary condition and the rest of the modules.

It turns out I already previously implemented support for unequal numbers of segments in the `Network` object. This seems to be the only change required.